### PR TITLE
tests(add-people): check names treated as HTML

### PR DESCRIPTION
**Pull request by OrionStar25**
These are tests for #1595 . I couldn't push to the original repo, so made a new PR.
Please review @marcoscaceres .

### CODE REVIEW

File: tests/spec/w3c/headers-spec.js [hunk 1]
```diff
@@ -276,6 +276,49 @@ describe("W3C — Headers", function() {
...
+          {
+            name:
+              "<span lang='ja'>阿南 康宏</span> (Yasuhiro Anan), (<span lang='ja'>第１版</span> 1st edition)",
+            company: "Microsoft",
+          },
+        ],
+      };
+      Object.assign(ops.config, newProps);
+      const doc = await makeRSDoc(ops);
+      var spanName = document.createElement("span");
```
Comment (marcoscaceres) [inline tests/spec/w3c/headers-spec.js]: Close, but here you need to reach into `doc` and find the `dd`.

Comment (marcoscaceres) [reply tests/spec/w3c/headers-spec.js]: See how to do it here: https://github.com/OrionStar25/respec/blob/5a9be94c046738954ac6867ad124753a25b153e4/tests/spec/w3c/headers-spec.js#L185

**Comment by OrionStar25** [comment]
@marcoscaceres the test for `formerEditors` is working but the same logic for `editors` isn't. Can you please help here?

**Comment by marcoscaceres** [comment]
Will take a look later today. Thanks for your efforts so far. Looks like it’s nearly there. We will need to fix up the commit history, but we can do that once the tests are working.

**Comment by OrionStar25** [comment]
Sure @marcoscaceres!

**Comment by marcoscaceres** [comment]
@OrionStar25, moved your tests here: https://github.com/w3c/respec/pull/1595/commits/1b5441efe88eb8fa9e4827962f8cf0de1e0727e1

Made some modifications to get everything working. Please take a look at the changes to understand what I changed. Happy to answer questions.

**Pull request closed without merging**